### PR TITLE
Grunt Toolchain

### PR DESCRIPTION
**Pull request by shirishgoyal**
**Toolchain**
- Run django server as grunt task
- SASS auto-compilation and source-map generation
- Live-reload for Frontend changes
- Flake8 auto-check for Python scripts

Updated documentation and fixed Flake8 issues

**Usage:**
Install npm dependencies (once)
`npm install`

Run web server
`grunt serve`

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [flake8]
-exclude = ./crowdsourcing/migrations,./crowdsourcing/tests
-ignore = E123,E128,E402,W503,E731,W601,F403
-max-line-length = 119
+exclude = ./crowdsourcing/migrations,./crowdsourcing/tests,./node_modules
+ignore = E123,E128,E133,E402,W503,E731,W601,F403
+max-line-length = 120
```
Comment (dmorina) [inline setup.cfg]: 119 is kinda the "new standard" instead of old 79 limit as specified by PEP

**Comment by dmorina** [comment]
Lgtm

**Comment by dmorina** [comment]
P.s did you want to add a pre-push hook for flake8 for those who don't want to use grunt

**Comment by shirishgoyal** [comment]
Pre-commit hook will mostly be added to documentation.
Will revert max-line length for PEP8.

**Comment by dmorina** [comment]
Sweet, thanks

**Pull request merged**